frontend/src/app/shared/components/grids/grid/page/grid-page.component.html

```diff
@@ -12,11 +12,12 @@ <h2 [textContent]="text.title"></h2>
       </ul>
   </div>
 
-  <div class="op-grid-page--grid-container" *ngIf="isTurboFrameSidebarEnabled()">
-    <div class="op-grid-page--main-content">
+  <!-- copied css classes from primer to have a UX behaviour consistent with other pages having a sidebar -->
+  <div class="Layout Layout--flowRow-until-md Layout--sidebarPosition-end Layout--sidebarPosition-flowRow-start" *ngIf="isTurboFrameSidebarEnabled()">
```

Comment:
The sidebar seems to take up a larger space than before on small screen. I'd suggest adding `Layout--sidebar-narrow` class, to make the width closer to the previous size.
```suggestion
  <div class="Layout Layout--flowRow-until-md Layout--sidebarPosition-end Layout--sidebarPosition-flowRow-start Layout--sidebar-narrow" *ngIf="isTurboFrameSidebarEnabled()">
```